### ISSUE TITLE
Package.swift need modify iOS deploy version 8 to 9

### ISSUE THREAD

**Issue by venice81** [RxSwiftCommunity/RxNimble#62]
When using SPM, the following error message is displayed.

```
The package product 'RxSwift' requires minimum platform version 9.0 for the iOS platform, but this target supports 8.0
The package product 'Nimble' requires minimum platform version 9.0 for the iOS platform, but this target supports 8.0
The package product 'RxTest' requires minimum platform version 9.0 for the iOS platform, but this target supports 8.0
The package product 'RxBlocking' requires minimum platform version 9.0 for the iOS platform, but this target supports 8.0
```

To solve this problem, you need to modify iOS deploy version macOS(.v10_10) to macOS(.v10_12) and iOS(.v8) to iOS(.v9) of Package.swift.

**Comment by timdolenko** [comment]
Pull request for that: #63

**Comment by ashfurrow** [comment]
This has been fixed in version 5.1.1, for both SPM and CocoaPods.
